src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -441,12 +441,12 @@ private DbContext TryCreateContextUsingAppCode(Type dbContextType, Type startupT
                 // Use EF design APIs to get the DBContext instance.
                 var operationHandler = new OperationReportHandler();
                 var operationReporter = new OperationReporter(operationHandler);
+                // EF infers the environment (Development/ Production) based on the environment variable
+                // ASPNETCORE_ENVIRONMENT. This should already be set up by the CodeGeneration.Design process.
                 var dbContextOperations = new DbContextOperations(
                     operationReporter,
                     dbContextType.GetTypeInfo().Assembly,
-                    startupType.GetTypeInfo().Assembly,
-                    "Development",
-                    Path.GetDirectoryName(_projectContext.ProjectFullPath));
```

Comment:
We want to expose a public API for this in the 2.0.0 timeframe, so we isolate you from additional breaks.